### DISCUSSION TITLE
Is it possible save default variable value when value is generated dynamically, but the variable text is static (fixed)?

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#48314] [Dashboards, Panels & Visualizations]
Is it possible save default variable value when value is generated dynamically, but the variable text is static (fixed)?

In my case I'd like to save variable with text (__text) is **current**

```SQL
select rand() as __value, 'value1' as __text union all
select rand() as __value, 'value2' as __text union all
select rand() as __value, 'current' as __text
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
